src/language-css/clean.js

```diff
@@ -198,13 +198,15 @@ function clean(original, cloned, parent) {
     }
   }
 
-  // We parse `@var[ foo ]` and `@var[foo]` differently
+  // We parse `@var[ foo ]` , `@var[foo]` and `var [ @foo ]` differently
```

Comment:
```suggestion
  // We parse `@var[ foo ]` , `@var[foo]`, and `var [ @foo ]` differently
```

Yup, we use the Oxford comma.